load_into_pg.py

```diff
@@ -295,22 +365,74 @@ def handleTable(table, insertJson, createFk, dbname, mbDbFile, mbHost, mbPort, m
 
 args = parser.parse_args()
 
-table = args.table
-
 try:
     # Python 2/3 compatibility
     input = raw_input
 except NameError:
     pass
 
+dbConnectionParam = buildConnectionString(args.dbname, args.host, args.port, args.username, args.password)
+
+# load given file in table
+if args.file and args.table:
+    table = args.table
+
+    if table == 'Posts':
+        # If the user has not explicitly asked for loading the body, we replace it with NULL
+        if not args.with_post_body:
+            specialRules[('Posts', 'Body')] = 'NULL'
+
+    choice = input('This will drop the {} table. Are you sure [y/n]?'.format(table))
+    if len(choice) > 0 and choice[0].lower() == 'y':
+        handleTable(table, args.insert_json, args.foreign_keys, args.file, dbConnectionParam)
+    else:
+        six.print_("Cancelled.")
+    if args.schema_name != 'public':
+        moveTableToSchema(table, args.schema_name, dbConnectionParam)
+    exit(0)
+
+# load a project
+elif args.so_project:
+    import libarchive
+
+    filepath = None
+    if args.file:
+        filepath = args.file
+    else:
+        # download the 7z archive in /tmp
+        file_name = args.so_project + '.stackexchange.com.7z'
+        url = '{0}/{1}'.format(args.archive_url, file_name)
+        filepath = '/tmp/'+file_name
```

Comment:
I think it would be better to use [tempfile](https://docs.python.org/3/library/tempfile.html) for creating a temporay directory instead of hardcoding '/tmp/' here. That will make the code OS independent and more robust.

Reply:
I didn't know about this library. Very good idea!